css/properties.json

```diff
@@ -2261,6 +2261,78 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border"
   },
+  "border-block": {
+    "syntax": "<'border-width'> || <'border-style'> || <'color'>",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Logical Properties"
+    ],
+    "initial": [
+      "border-top-width",
+      "border-top-style",
+      "border-top-color"
+    ],
+    "appliesto": "allElements",
+    "computed": [
+      "border-top-width",
+      "border-top-style",
+      "border-top-color"
+    ],
+    "order": "uniqueOrder",
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-block"
+  },
+  "border-block-color": {
```

Comment:
Would it make more sense for the syntax to be `<'color'>{1,2}`, as you can specify one or two colors — both the same or separate colors for each, can't you? And same pattern for `*-style` and `*-width`?

Also same pattern for the inline versions?

Reply:
so it looks like the spec isn't consistent with these. These ones use border-top-color, the border-block border-inline properties just use color. I've updated everything to match the spec even if that seems inconsistent.